### DISCUSSION TITLE
In memory caching of frequevently used tables? Question

### DISCUSSION THREAD

**Discussion by JDRanpariya** [trinodb/trino#14097] [Q&A] [answered]
is there a way to cache in-memory with Rubix or Alluxio? I'm looking for in-memory caching of frequently used tables. is it possible with trino in-memory or apache ignite?

**Comment by raunaqmorarka** [comment, marked as answer]
You can cache data in-memory with Rubix by setting up a ramdisk and providing a path to that in configs.
With Alluxio it's best to ask the Alluxio community about the possibilities.
Trino also has a memory connector that can be used to keep specific tables in-memory, although creation and updates to such tables has to be done manually.
Caching in-memory on the Trino cluster has the potential to interfere with memory usage during query execution unless accounted for properly.
There is an open PR for Apache Ignite integration https://github.com/trinodb/trino/pull/8323, but it appears to be stalled.
